README.md

```diff
@@ -1,20 +1,133 @@
 
 # customs-data-store
 
+[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0) [![Coverage](https://img.shields.io/badge/test_coverage-90-green.svg)](/target/scala-3.3.5/scoverage-report/index.html) [![Accessibility](https://img.shields.io/badge/WCAG2.2-AA-purple.svg)](https://www.gov.uk/service-manual/helping-people-to-use-your-service/understanding-wcag)
+
 This repository contains the code for a persistent cache holding customs related data.
```

Comment:
Better to add below line as well

This Microservice is a common backend service and used by other CDS teams (Exports and Reimbursement) as well.